### DISCUSSION TITLE
Manual deployment to IAT

### DISCUSSION THREAD

**Discussion by nicward** [microsoft/AL-Go#535] [Q&A] [answered]
Hello,

Sorry, more questions.  Is it possible to have continuous deployment to a DEV sandboxes and manual deployment to UAT and Production?

We have a lot of integrations in out systems and we need to test these in DEV before moving to UAT.  I can see that it would be quite easy to change the settings so continuous deployment is to DEV instead of UAT as it's just changing the environment names.  However, setting up two workflows for UAT & DEV seems more tricky.    

I'm sure it's possible but could do with some pointers on how to set this up?

Thanks

Nick

**Comment by freddydk** [comment]
At this time, the only thing determining whether an environment is selected for continuous deployment or manual deployment is the TAG - like (PROD).
We will add a mechanism to set this in settings.

**Comment by freddydk** [comment, marked as answer]
In the latest version, you can define which environments are used for Continuous Deployment using the DeployTo settings structure.

```
"DeployToUAT": {
    "EnvironmentName": "RealName",
    "ContinuousDeployment": false
}
``` 

If not specified, Continuous Deployment is derived from the name like before.
